data/immutability.js

```diff
@@ -0,0 +1,188 @@
+      { text: _("Enlightenment"), correct: false },
+    ],
+  },
+  {
+    text: _("What is the name of installer in openSUSE Aeon?"),
+    answers: [
+      { text: _("tik"), correct: true, },
+      { text: _("tak"), correct: false },
+      { text: _("Anaconda"), correct: false },
+      { text: _("Agama"), correct: false },
+    ],
+  },
+  {
+    text: _("What bootloader is used by default in openSUSE Aeon?"),
+    answers: [
+      { text: _("lilo"), correct: false },
+      { text: _("systemd-boot"), correct: true,},
+      { text: _("grub2"), correct: false },
+      { text: _("BOOTMGR"), correct: false },
+    ],
+  },
+  {
+    text: _("What do Aeon and Kalpa have in common?"),
+    answers: [
+      { text: _("Both are based on MicroOS with a ready-to-use desktop."), correct: false },
+      { text: _("Both use transactional-updates for rpm installation."), correct: false },
+      { text: _("Both promote distrobox and flatpaks for extra apps."), correct: false },
+      { text: _("All options are correct."), correct: true },
+    ],
+  },
+  {
+    text: _("Both Aeon and Kalpa have interesting names indirectly referring to a rolling release. How so?"),
+    answers: [
+      {
+        text: _("Both are ancient terms meaning long periods or cosmic cycles."),
+        correct: true,
+      },
+      {
+        text: _("Both are names of famous Western movies where rolling tumbleweed appeared."),
+        correct: false,
+      },
+      {
+        text: _("Both are very old nicknames of the original architect behind MicroOS."),
+        correct: false,
+      },
+      {
+        text: _("Both refer to software update tools in openSUSE."),
+        correct: false,
+      },
+    ],
+  },
+  {
+    text: _("There was an experiment with a Sway-based immutable desktop in 2023. Do you know its name?"),
+    answers: [
+      { text: _("Greyhound"), correct: false },
+      { text: _("Greybeard"), correct: true },
+      { text: _("Bluehair"), correct: false },
+      { text: _("Ultraviolet"), correct: false },
+    ],
+  },
+  {
+    text: _("Which tool is used to safely update the base system in our Immutable distributions?"),
+    answers: [
+      { text: _("zypper dup"), correct: false },
+      { text: _("dnf upgrade"), correct: false },
+      { text: _("transactional-update"), correct: true },
+      { text: _("ostree pull"), correct: false },
+    ],
+  },
+  {
+    text: _("How can you install an additional package?"),
+    answers: [
+      { text: _("zypper in"), correct: false },
+      { text: _("dnf install"), correct: false },
+      { text: _("transactional-update pkg install"), correct: true },
+      { text: _("rpm-ostree install"), correct: false },
+    ],
+  },
+  {
+    text: _("Why is 'transactional-update shell' generally not recommended?"),
+    answers: [
+      { text: _("While it can be helpful, this command should ideally never be used. It can be dangerous and brick the system."), correct: true },
```

Comment:
I think the wording is a bit too strong here. How about adding a `Because you can modify all parts of the system ` or `If used incorrectly ` as the beginning of the last sentence.

Reply:
I think  simply taking on "Because you can modify all parts of the system" will work well. People will get it.